pep/ember-cli-build.js

```diff
@@ -5,7 +5,8 @@ const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 module.exports = function(defaults) {
     let app = new EmberApp(defaults, {
         fingerprint: {
-            prepend: process.env.ASSETS_BASE_URL
+            prepend: process.env.ASSETS_BASE_URL,
+            extensions: ['js', 'css', 'png', 'jpg', 'gif', 'map', 'xslt']
```

Comment:
did you test if adding the extension here alone will fingerprint the file in a prod build? I thought that ember doesn't auto-fingerprint dynamically generated paths that are in JS-code, only paths in templates, and you needed to use a specific ember-cli api to get the fingerprinted url in those cases. I could be wrong on that tho.